### PR TITLE
Release notes for Percona XtraDB Cluster 5.6.26-25.12

### PR DESCRIPTION
**Pull request by hrvojem**

### CODE REVIEW

**Comment by ronin13** [comment]
https://jira.percona.com/browse/PXC-447
https://jira.percona.com/browse/PXC-448
https://jira.percona.com/browse/PXC-445

need to be added to RN as well.

**Comment by kbauskar** [comment]
LGTM

File: percona-xtradb-cluster/5.6/source/release-notes/Percona-XtraDB-Cluster-5.6.26-25.12.rst [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ Variable :variable:`wsrep_dirty_reads` now has Global scope as well. (:bug:`1505609`). 
+
+Bugs Fixed
+==========
+
+ Asynchronous replication slave thread would continue to run and execute even if the asynchronous slave was non-primary. Bug fixed :wsrepbug:`188`.
+
+ If a cluster conflict happened while asynchronous replication was running it would cause the following message to be printed repeatedly in the error log: ``conflict state 7 after post commit``. Bug fixed :wsrepbug:`181`.
+
+ If a transaction was Brute-Force aborted, subsequent ``Performance Schema`` statement would either cause the client to assert or fail to return a result set. Bug fixed :wsrepbug:`126`.
```
Comment (ronin13) [inline percona-xtradb-cluster/5.6/source/release-notes/Percona-XtraDB-Cluster-5.6.26-25.12.rst]: PS here is prepared statement.

File: percona-xtradb-cluster/5.6/source/release-notes/Percona-XtraDB-Cluster-5.6.26-25.12.rst [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+ Server assertion would happen when the ``Perfomance Schema`` instrumentation from a previous statement was not correctly reset before the next one or if a statement could not be certified because one of the nodes did not respond. Bug fixed :bug:`125`.
```
Comment (ronin13) [inline percona-xtradb-cluster/5.6/source/release-notes/Percona-XtraDB-Cluster-5.6.26-25.12.rst]: This also hit with prepared statements, but Perf Schema part is also required here.

**Pull request merged**